plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergErrorCode.java

```diff
@@ -24,7 +24,7 @@
 public enum IcebergErrorCode
         implements ErrorCodeSupplier
 {
-    ICEBERG_UNKNOWN_TABLE_TYPE(0, EXTERNAL),
+    // code ICEBERG_UNKNOWN_TABLE_TYPE(0, EXTERNAL) is deprecated
```

Comment:
simply remove?

Reply:
I'm following the pattern `code XXX(YY) is deprecated` which is used in other `...ErrorCode` classes.